### PR TITLE
Effects VB integration doc

### PR DESCRIPTION
**Pull request by KaustubhKumar05**

### CODE REVIEW

File: docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+```bash section=GetHMSVirtualBackgroundPackage sectionIndex=1
+npm install --save @100mslive/hms-virtual-background@latest
+```
+
+## Features
+
+The following features are currently supported under the `HMSEffectsPlugin` class:
+
+```js
+setBlur(blurPower: number)
```
Comment (raviteja83) [inline docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx:25]: setPreset is missing here I think

Comment (raviteja83) [reply docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx:25]: same with getPreset as  well

Comment (KaustubhKumar05) [reply docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx]: Updated

File: docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx [hunk 2]
```diff
@@ -22,27 +22,38 @@ npm install --save @100mslive/hms-virtual-background@latest
...
 */
+getBackground()
+
+
+/*
+  Can be used to set the quality of VB. The default is "balanced"
+  Options: "lightning" | "speed" | "balanced" | "quality"
+  The quality of the VB and CPU usage increases from left to right
+ */
+setPreset()
```
Comment (raviteja83) [inline docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx:47]: should have an argument and return type right? why are there no return types on all functions

Comment (KaustubhKumar05) [reply docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx]: Updated

File: docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+    over <a href={'https://discord.com/invite/kGdmszyzq2'}>Discord</a>.
+</Note>
+
+
+## Pre-requisites
+
+**Get the 100ms VirtualBackground Package**
+
+```bash section=GetHMSVirtualBackgroundPackage sectionIndex=1
+npm install --save @100mslive/hms-virtual-background@latest
```
Comment (raviteja83) [inline docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx:17]: Let us also mention the version from which this is supported

File: docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+  const hmsActions = useHMSActions();
+  // Get the effects SDK key here
+  const effectsKey = useHMSStore(selectEffectsKey);
+  const trackId = useHMSStore(selectLocalVideoTrackID);
+    
+  // State can be used to show active selection
+  const [background, setBackground] = useState<string | HMSVirtualBackgroundTypes>(
+      VBHandler.getBackground() as string | HMSVirtualBackgroundTypes,
+  );
+
```
Comment (raviteja83) [inline docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx:157]: formatting is not proper

File: docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx (hunk 3 above)
Comment (KaustubhKumar05) [reply docs/javascript/v2/how-to-guides/extend-capabilities/plugins/effects-virtual-background.mdx:17]: Added

**Pull request merged**